src/gss_spi.c

```diff
@@ -452,3 +452,12 @@ OM_uint32 gssspi_mech_invoke(OM_uint32 *minor_status,
     return gssntlm_mech_invoke(minor_status, desired_mech, desired_object,
                                value);
 }
+
+OM_uint32 gss_set_cred_option(OM_uint32 *minor_status,
```

Comment:
Presumably this should be `gssspi_set_cred_option`? (Note that the `cred_handle` has different level of pointer indirection on that method!)

https://github.com/krb5/krb5/blob/2b34a007461065e0cab4490dfe1ae5ddd10da67b/src/lib/gssapi/mechglue/g_initialize.c#L745

Reply:
gah, yeah, my tests call the internal functions directly and did not catch this ...
I knew asking for your review would be fruitful :-D

Reply:
Hopefully fixed now

Reply:
> I knew asking for your review would be fruitful :-D

Always happy to find obscure bugs in other people's security/cryptography/authentication libraries.

> Hopefully fixed now

Yep, works great now.